### DISCUSSION TITLE
在网站嵌入 B 站的视频，会显示的很宽 ，高度很小

### DISCUSSION THREAD

**Discussion by wgx-dh** [YunYouJun/hexo-theme-starry#349] [Q&A 疑难杂症]
![image](https://user-images.githubusercontent.com/63238251/191706583-ae057a15-51dd-49c4-b192-8e934e3ef796.png)

**Comment by YunYouJun** [comment]
<img width="1116" alt="image" src="https://user-images.githubusercontent.com/25154432/191794264-e9fc262c-4ab8-4316-956f-f2a47e0b5f27.png">

考虑到 iframe 内容可能各不相同，没办法有一个统一的样式。
给内嵌 bilibili 视频的场景添加了一个辅助类：`.bilibili-video`。为 `iframe` 添加 `class="bilibili-video"`。

```html
<iframe class="bilibili-video" src="//player.bilibili.com/player.html?aid=626900451&bvid=BV17t4y1S7tz&cid=229255224&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```

Released in `v1.10.8`.
